README.md

```diff
@@ -9,7 +9,21 @@ current region or entire buffer.
 
 Download the package and add it to your load path.
 
-(Hopefully MELPA coming soon.)
+To install from [MELPA](https://melpa.org/), add the following to your `.emacs` or `.emacs.d/init.el`:
```

Comment:
It might be worth linking to the MELPA setup docs at https://github.com/melpa/melpa#usage rather than repeating them here.